modules/gitops-deploy-resources-different-namespaces.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="deploy-resources-different-namespaces_{context}"]
+= Deploy resources to a different namespace
+
+[role="_abstract"]
+To allow the Argo CD to manage resources in other namespaces apart from where it is installed, configure the target namespace with a label "argocd.argoproj.io/managed-by".
+
+.Procedure
+. Run the following command to configure the namespace.
++
+[source,terminal]
+----
+$ oc label namespace <name_of_namespace> argocd.argoproj.io/managed-by=<instance_name> \\ the instance name is the name of the namespace where the instance is installed.
```

Comment:
```suggestion
$ oc label namespace <name_of_namespace> && \
argocd.argoproj.io/managed-by=<instance_name>
```
- Avoid comments in command line examples. You can put this information in the sentence that precedes the command or in a callout after it.
- Optionally, you can wrap long commands by using `&& \`

Reply:
If needed, you can use callouts to give more information about  user-replaceable values like these. For example:
```
----
$ oc label namespace <name_of_namespace> && \
argocd.argoproj.io/managed-by=<instance_name> <1>
----
<1>  The `instance_name` is the name of the namespace where the instance is installed.
```